engine/api/application.go

```diff
@@ -66,9 +66,14 @@ func (api *API) getApplicationsHandler() service.Handler {
 			}
 		}
 
-		applications, err := application.LoadAll(api.mustDB(), projectKey, loadOpts...)
+		proj, err := project.Load(api.mustDB(), projectKey)
```

Comment:
I think it's a bit sad to load the project to get its ID, whereas the projectKey is an identifier.
we should consider `application.LoadAllByProjectKey`